public/components/datasources/components/manage/accelerations/utils/acceleration_utils.tsx

```diff
@@ -80,9 +81,11 @@ export const onRefreshButtonClick = (acceleration: any) => {
   console.log('refreshing', acceleration.name);
 };
 
-export const onDiscoverButtonClick = (acceleration: any) => {
-  // TODO: send user to Discover
-  console.log('sending user to discover for', acceleration.name);
+export const onDiscoverButtonClick = (acceleration: CachedAcceleration) => {
+  // boolean determining whether its a skipping index table or mv/ci
+  if (acceleration.type === 'skipping' || acceleration.type === undefined) return;
```

Comment:
Why is skipping index a no-op here. If the discover button is click for a skipping index shouldn't load a direct query to table in explorer? 

```
source = ds.db.table | head 10 
```

Reply:
The only places where this method specifically is called are `accelerations_details_flyout.tsx` and `acceleration_table.tsx`. The flyout shouldn't be opened for skipping indices and the table shouldn't contain any skipping indices. This is more of a check rather than a no-op.

Reply:
I don't think that's the case for the accelerations. We have the capability to open a skipping index flyout here.

Reply:
Fixed!